webapp/src/Service/RejudgingService.php

```diff
@@ -99,43 +100,62 @@ public function createRejudging(
             }
 
 
-            $this->em->wrapInTransaction(function () use (
```

Comment:
Did you check that this one was also slow?

If I understand the change correctly we still do a transaction but would loose the flush to the entitymanager, wouldn't we need the flush after the last transaction. Wouldn't we want all those transactions in 1 go?

Reply:
This made it go from ~3x to ~50x speed improvement. The wrap does the exact same thing AND a flush. We don't need the flush if we don't use Doctrine and only direct queries, which we now do.

Doing one transaction might be faster but we also lock the database for some operations for the duration of the transaction, so we better keep it short. I don't think it will make a very big difference anyway.